.gitpod.yml

```diff
@@ -0,0 +1,9 @@
+image:
+  file: .docker/gitpod/mathlib/Dockerfile
+
+vscode:
+  extensions:
+    - jroesch.lean
```

Comment:
This seems to work fine without pinning a version number, @gebner.

Reply:
There's some possibility that behind the scenes it is slower, however, if this causes it to look it up everytime rather than pulling it out of some cache.